### ISSUE TITLE
Sample display messed up when collection is shared

### ISSUE THREAD

**Issue by schatzsc** [ComPlat/chemotion_ELN#1599]
When a coworker shared her collection with me for reviewing, the sample list display is seriously messed up as shown in the top figure attached while the associated reaction list below displays correctly. The compounds in the "sample list" are the same as used in the reactions, therefore we suspect it might have something to do with the complicated structures and possibly a corrupted molfile passed on between different parts of Chemotion?!? Also not sure how to provide further information for debugging - would molfiles of the structures/compounds be helpful?

![Chemotion sharing](https://github.com/ComPlat/chemotion_ELN/assets/88166560/b8cc4c95-1d2e-4487-832c-f02634fdf5c0)
![Chemotion sharing2](https://github.com/ComPlat/chemotion_ELN/assets/88166560/5a2fb822-8afc-44a3-b179-c5f8ba17f490)

**Comment by PiTrem** [comment]
Hi @schatzsc 

could the user check the permissions set on the collection ?

![image](https://github.com/ComPlat/chemotion_ELN/assets/15799181/5c0877f0-a5b7-4e34-a321-e6ef79b7e9e6)

**Comment by schatzsc** [comment]
We can do this next Monday when we get together for the weekly group seminar, but nothing was changed relative to the standard installation, at least on the level of what's accessible to the regular user.

**Comment by schatzsc** [comment]
It seems like this problem appears when you share some samples and/or reaction more than one time with the same person and with different permissions.

When we adjusted all permissions of these "multile sharings" or deleted all shared collections and then re-assigned the samples and reaction to a new shared collection, it looks like the problem has disappeared?!? Still applies to version 1.7.2

**Comment by nicolejung** [comment]
If you share a sample with different permissions levels, then the most restrictive should be valid - to ensure that only the desired information is visible - and not shared accidentally. Only if the other person has all necessary rights (on the relevant details) the information is visible. Invisible samples are missing the image and further structure information (only the sum formula is provided).

**Comment by schatzsc** [comment]
That's correct and I will close the issue if there are not further comments.

What mostly confused us was the fact that it seems like you can share a certain collection or parts of a collection multiple times with the same person but with different rights levels.

Would it be possible to "homogenize" this in a "clean-up" function, which combines the different sharings?

Possibly not and the problem appeared when the student added more compounds and reactions to Chemotion and then also wanted to shared this with me together with entries already shared previously.

Still, it seems to be a somewhat error-prone procedure to me ...
